scout/parse/omim.py

```diff
@@ -349,7 +349,7 @@ def get_mim_genes(genemap_lines, mim2gene_lines):
     return hgnc_genes
 
 
-def get_mim_phenotypes(genemap_lines: Iterable[str]) -> Dict[int, Any]:
+def get_mim_phenotypes(genemap_lines: Iterable[str]) -> Dict[str, Any]:
```

Comment:
I guess you are modifying the keys according to the format of the other file where disease_id, i.e. OMIM:91921. Fine if it works!

Reply:
Yes, and it also feels a bit risky to use only the number as key since this might result in identical keys between ORPHA terms and OMIM terms later.